src/style.css

```diff
@@ -1,16 +1,34 @@
 html {
-  font-family: Cambria, Cochin, Georgia, Times, "Times New Roman", serif;
+  box-sizing: border-box;
+  font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
+}
+
+body {
+  height: auto;
+  box-shadow: 2px 2px 2px 2px #000;
+  border-radius: 5px;
+  width: 90%;
+  margin: 20px auto 10px auto;
 }
 
 main {
   padding: 20px;
-  width: 80%;
-  margin: 0 auto;
+  width: 82%;
+  height: 100vh;
+}
+
+#add-new-game {
+  height: auto;
+  width: 100%;
+  box-shadow: 2px 2px 2px 2px #000;
+  padding: 10px;
 }
 
 .rec-scores {
-  float: left;
-  width: 40%;
+  width: 100%;
+  display: none;
+  margin-top: 10px;
+  padding-bottom: 20px;
```

Comment:
- [ ] Kindly use two ✌🏽 columns for the main content as in the wireframe, it is because if I had a long list of thousand scores I should scroll down until I find the form to add a new score. I suggest using `overflow-y: scroll;` in the score list for a better UI. 👍🏽